app/assets/stylesheets/card.scss

```diff
@@ -33,7 +35,7 @@ body {
   }
 
   a {
-    color: #ffab40;
+    color: #29A68F;
```

Comment:
Color literals like `#29A68F` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#29A68F` should be written as `#29a68f`